liblangutil/ErrorReporter.cpp

```diff
@@ -28,6 +28,20 @@ using namespace std;
 using namespace solidity;
 using namespace solidity::langutil;
 
+namespace solidity::langutil {
```

Comment:
Usually we use explicit namespace prefixes if the reference is ambiguous instead of opening namespaces in cpp files.